src/components/10-atoms/toggle-switch/react/ControlledWithDummyOnChange.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import AXAToggleSwitchReact from './AXAToggleSwitchReact';
+
+export default class example extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      active: true,
+      receivedEvent: '-',
+    };
+  }
+
+  change = ev => {
+    this.setState({ active: ev.target.checked });
+    this.setState({ receivedEvent: String(ev.target.checked) });
```

Comment:
I think you can combine this into a single setState... (but see below, maybe active not even needed)